go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -369,7 +370,10 @@ func (bnc *BaseNetworkController) createNodeLogicalSwitch(nodeName string, hostS
 		Name:      types.SwitchToRouterPrefix + switchName,
 		Type:      "router",
 		Addresses: []string{"router"},
-		Options:   map[string]string{"router-port": types.RouterToSwitchPrefix + switchName},
+		Options: map[string]string{
+			"router-port": types.RouterToSwitchPrefix + switchName,
+			"arp_proxy":   kubevirt.ComposeARPProxyLSPOption(),
+		},
```

Comment:
maybe I'm missing something, but I'm a bit surprised to see this being used **always**.

Reply:
The arp_proxy at OVN has less priority than the switch so it do not hurt, also it's kind of redundant for to known IPs since it will return a mac for the router LSP port.